### ISSUE TITLE
Unable to totally disable Google Assist on brand new Android 6.0.1

### ISSUE THREAD

**Issue by UTEMScripts** [HairyRobot/SwipeLaunchDisabler#1]
I've used this most excellent SwipeLaunch Disabler for nearly four years.  Just got a new phone that runs Android 6.0.1.  I followed your special instructions for Marshmallow, but I can't seem to get the swipe launch to do anything other than launch the screen shown here.  Perhaps it is worth mentioning that I don't see a "none" option in the Assist & Voice Input menu (screenshow also attached).  Any advice?

![screenshot_20180314-114535](https://user-images.githubusercontent.com/13245062/37417438-5295bdb6-277e-11e8-9438-0ce0472a3809.jpg)

![screenshot_20180314-114847](https://user-images.githubusercontent.com/13245062/37417472-656887b6-277e-11e8-8dc2-27b9a8f52ff4.jpg)

**Comment by HairyRobot** [comment]
The exact location of the options may differ between Android version.

Referring to the screenshot-20180314-114847, have you try tapping on the
"cog wheel" icon next to the "Assist app" option, and see what further
options the system will display.

Thanks.

**Comment by UTEMScripts** [comment]
I swear that yesterday I clicked on that cog multiple times and was brought to a menu that had some options related to Google Assist.  Just now I clicked on and was prompted with a pop-up asking which app I want to use as my default assist app (see screen shot).  Now my problem is resolved.  I don't know what changed between yesterday and today that led to the different behavior.  In any case, thank you for this app and for your assitance.

![Assist App Cog PopUp](https://user-images.githubusercontent.com/13245062/37472193-7b8de386-2839-11e8-88d3-de7d26928134.png)
